packages/integration-tests/tests/__snapshots__/v1014-multi-project-manual-config.test.ts.snap

```diff
@@ -2,145 +2,5 @@
-  366:1  error  This line has a length of 176. Maximum allowed is 140   max-len
-  379:1  error  This line has a length of 167. Maximum allowed is 140   max-len
-  385:1  error  This line has a length of 167. Maximum allowed is 140   max-len
-  391:1  error  This line has a length of 167. Maximum allowed is 140   max-len
-  397:1  error  This line has a length of 165. Maximum allowed is 140   max-len
-  403:1  error  This line has a length of 165. Maximum allowed is 140   max-len
-  409:1  error  This line has a length of 165. Maximum allowed is 140   max-len
-  429:1  error  This line has a length of 188. Maximum allowed is 140   max-len
-  430:1  error  This line has a length of 158. Maximum allowed is 140   max-len
-  431:1  error  This line has a length of 165. Maximum allowed is 140   max-len
-  432:1  error  This line has a length of 165. Maximum allowed is 140   max-len
-  433:1  error  This line has a length of 162. Maximum allowed is 140   max-len
-  440:1  error  This line has a length of 187. Maximum allowed is 140   max-len
-  441:1  error  This line has a length of 159. Maximum allowed is 140   max-len
-  442:1  error  This line has a length of 188. Maximum allowed is 140   max-len
-  443:1  error  This line has a length of 174. Maximum allowed is 140   max-len
-  450:1  error  This line has a length of 172. Maximum allowed is 140   max-len
-  457:1  error  This line has a length of 190. Maximum allowed is 140   max-len
-  458:1  error  This line has a length of 160. Maximum allowed is 140   max-len
-  462:1  error  This line has a length of 217. Maximum allowed is 140   max-len
-  466:1  error  This line has a length of 187. Maximum allowed is 140   max-len
-  474:1  error  This line has a length of 190. Maximum allowed is 140   max-len
-  475:1  error  This line has a length of 160. Maximum allowed is 140   max-len
-  476:1  error  This line has a length of 410. Maximum allowed is 140   max-len
-  483:1  error  This line has a length of 2884. Maximum allowed is 140  max-len
-  509:1  error  This line has a length of 261. Maximum allowed is 140   max-len
-  514:1  error  This line has a length of 230. Maximum allowed is 140   max-len
-  518:1  error  This line has a length of 148. Maximum allowed is 140   max-len
-  519:1  error  This line has a length of 1046. Maximum allowed is 140  max-len
-
-✖ 41 problems (41 errors, 0 warnings)
-
-
-Linting \\"horse-plants\\"...
-All files pass linting.
-
-
-Linting \\"allergies-fusion\\"...
-All files pass linting.
-
-
-Linting \\"breakfast-cat\\"...
-All files pass linting.
-
-
-Linting \\"prints-soda\\"...
-All files pass linting.
-
-
-Linting \\"ice-cream-monster\\"...
-All files pass linting.
-
-
-Linting \\"shoe-prints\\"...
-All files pass linting.
-
-
-Linting \\"websites-system\\"...
-All files pass linting.
-
-
-Linting \\"robot-websites\\"...
-All files pass linting.
-"
+Linting \\"v1014-multi-project-manual-config\\"..."
```

Comment:
This snapshot is clearly broken, you can debug by cd'ing into the relevant workspace in the integration-tests package fixtures and running normal `ng lint` commands

Reply:
I did and I discovered that the problem is that the ng lint is not running for this folder (_v1014-multi-project-manual-config_). When I try to run it, I get the following error:


```
[error] Error: Failed to load plugin '@angular-eslint' declared in '.eslintrc.json#overrides[0]': Cannot find module '/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@angular-eslint/eslint-plugin/dist/index.js'. Please verify that the package.json has a valid "main" entry
Require stack:
- /PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/__placeholder__.js
Referenced from: /PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/.eslintrc.json
    at tryPackage (node:internal/modules/cjs/loader:338:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:551:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:905:27)
    at Function.resolve (node:internal/modules/cjs/helpers:98:19)
    at Object.resolve (/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@eslint/eslintrc/lib/shared/relative-module-resolver.js:28:50)
    at ConfigArrayFactory._loadPlugin (/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@eslint/eslintrc/lib/config-array-factory.js:1011:39)
    at ConfigArrayFactory._loadExtendedPluginConfig (/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@eslint/eslintrc/lib/config-array-factory.js:831:29)
    at ConfigArrayFactory._loadExtends (/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@eslint/eslintrc/lib/config-array-factory.js:778:29)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/PATH/angular-eslint/packages/integration-tests/fixtures/v1014-multi-project-manual-config/node_modules/@eslint/eslintrc/lib/config-array-factory.js:719:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
```

I also noticed that this happens only for this fixture, the others ran normally.

Reply:
@JamesHenry I would appreciate your help on this to move forward with this PR, thanks.